packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -177,7 +184,7 @@ The following customizations aren't supported:
 * **Okta-hosted widget:** The **Help title** link in the **Customized Help Links** section of the Customization page has been removed and isn't supported.
```

Comment:
Line 168 on `features.showPasswordToggleOnSignInPage` I'd remove the last sentence about what we're adding in a future release.